gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -105,9 +109,50 @@ func (mappingTransform *mappingExporter) TransformCommand(ctx context.Context, i
 		mappingTransform.thread = inputCommands[0].Thread()
 	}
 
+	for _, cmd := range inputCommands {
+		if queueSubmit, ok := cmd.(*VkQueueSubmit); ok {
+			if err := mappingTransform.recordSubmittedRenderPass(ctx, queueSubmit, id.GetID(), inputState); err != nil {
+				return nil, err
+			}
+		}
+	}
+
 	return inputCommands, nil
 }
 
+func (mappingTransform *mappingExporter) recordSubmittedRenderPass(ctx context.Context, cmd *VkQueueSubmit, cmdID api.CmdID, inputState *api.GlobalState) error {
```

Comment:
The function is relatively long and boiler-plate-y. A short function comment would help me as a reader to quickly understand what it is supposed to do without having to parse and interpret the whole function body.

If I understand correctly, it just populates the `usedFramebuffers` map?

Reply:
Well, it literally records the renderpass that has been actually submitted to GPU. I think one thing to clear out is that it tracks the Framebuffer handle and not the Renderpass handle.